### PR TITLE
Fix for #133

### PR DESCRIPTION
**Pull request by pranavq212**
https://github.com/exupero/saveSvgAsPng/issues/133

In chrome, download doesnt happen for more than 1.5MB png file 

Now work for all browser seemlessly

### CODE REVIEW

File: saveSvgAsPng.js [hunk 1]
```diff
@@ -418,6 +418,14 @@
         saveLink.href = uri;
```
Comment (guilhermesimoes) [inline saveSvgAsPng.js:418]: Isn't this line useless due to line 423?

File: saveSvgAsPng.js [hunk 2]
```diff
@@ -418,6 +418,14 @@
...
         saveLink.style.display = 'none';
         document.body.appendChild(saveLink);
+        var blob = uriToBlob(uri);
+        var url = URL.createObjectURL(blob);
+        saveLink.href = url;
+        saveLink.onclick = function() {
+          requestAnimationFrame(function() {
+            URL.revokeObjectURL(url);
+          })
+        };
```
Comment (guilhermesimoes) [inline saveSvgAsPng.js:428]: What is this code doing? 😄

**Comment by houshuang** [comment]
This works great for us, and I hope it could get merged. We are letting users export large SVGs which cannot be fully seen on screen (for screenshots), because users scroll. This also means file-size becomes a problem.

**Comment by exupero** [comment]
Thanks for confirmation that it works. Merging.

**Comment by exupero** [comment]
I've wrapped this code in a try/catch since `URL.createObjectURL` and `URL.revokeObjectURL` are documented as experimental technology. If it fails, there'll be a warning in the console, and the code will fall back to the old method. See 972164f61e6997182a0e5aa799c46d7f79778787.

These changes are on NPM as version 1.2.0. Thanks, @pranavq212! Sorry it took so long.

**Pull request merged**